docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,368 @@
+---
+title: "Deploying a High Availability Teleport Cluster"
+description: "Deploying a High Availability Teleport Cluster"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure during an incident in
+your Teleport cluster. In this guide, we will explain the components of a
+high-availability Teleport deployment.
+
+## Overview
+
+A high-availability Teleport cluster revolves around a group of redundant
+`teleport` processes, each of which runs the Auth Service and Proxy Service,
+plus the infrastructure required to support them. 
+
+This includes:
+
+- **A Layer Four load balancer** to direct traffic from users and services to an
+  available `teleport` process.
+- A **key-value store** for Auth Service state and audit events that all Auth
+  Service instances can access. This requires permissions for Auth Service
+  instances to manage records within the key-value store.
+- An **object storage service** for session recordings uploaded by the Auth
+  Service. This requires permissions for `teleport` instances to manage objects
+  within the storage service.
+- An **automated TLS credential provisioner** that obtains TLS credentials from
+  Let's Encrypt, renews the credentials, and provisions `teleport` instances
+  with them.
+- A **DNS service** you can use to create records for the Teleport Proxy
+  Service. The TLS credential provisioner will also use this to create records
+  to demonstrate control over your domain name and fetch credentials from Let's
+  Encrypt.
+
+{/* TODO: include a diagram--do this after filling in the rest of the details*/}
+
+## Layer Four load balancer
+
+The load balancer forwards traffic from users and services to an available
+Teleport instance. This must not terminate TLS, and must transparently forward
+the TCP traffic it receives. In other words, this must be a Layer Four load
+balancer, not a Layer Seven (e.g., HTTP) load balancer. 
+
+Configure the load balancer to forward traffic from the following ports on the
+load balancer to the corresponding port on an available Teleport instance:
+
+| Port | Description |
+| - | - |
+| `3023` | SSH port for clients connect to. The Proxy Service will forward this connection to port `3022` on the destination Node. |
```

Comment:
When you're connecting to a node joined over a reverse tunnel, port 3022 isn't used at all. The proxy routes your connection to the correct reverse tunnel and provides you with a transport channel to the node which flows over that reverse tunnel.